### PR TITLE
Extra Mime Emotes

### PR DESCRIPTION
**Pull request by ghost**
I speak for the mimes.

:cl:
 * rscadd: Mime lines for sigh, chuckle, grumble, mumble, gasp, gag, and burp.

### CODE REVIEW

File: code/modules/mob/living/carbon/emote.dm [hunk 1]
```diff
@@ -220,6 +223,7 @@
 	key = "chuckle"
 	key_third_person = "chuckles"
 	message = "chuckles."
+	message_mime = "puts a hand on their chest and imitates a smug chuckle."
```
Comment (DamianX) [inline code/modules/mob/living/carbon/emote.dm]: i have no hands

Comment (boy2mantwicethefam) [reply code/modules/mob/living/carbon/emote.dm]: I do

Comment (ghost) [reply code/modules/mob/living/carbon/emote.dm]: You can't use that excuse anymore.

Comment (DamianX) [reply code/modules/mob/living/carbon/emote.dm]: no really, you can't assume the mime's hands haven't been removed

Comment (ghost) [reply code/modules/mob/living/carbon/emote.dm]: I know, thanks for pointing it out (you arm-having freak)

**Pull request merged**